client/src/components/UI/__tests__/navBar.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import NavBar from '../NavBar';
+
+describe('Tests for <NavBar />', () => {
```

Comment:
In `describe()`, just write `<NavBar />` rather than `Test for <NavBar />`. Will look better and more intuitive while running the tests.

[Example](https://github.com/airbnb/enzyme#shallow-rendering)